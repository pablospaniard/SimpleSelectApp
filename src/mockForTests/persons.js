export const persons = [
  {
    gender: 'female',
    name: {
      title: 'ms',
      first: 'nicky',
      last: 'vargas'
    },
    location: {
      street: '9761 station road',
      city: 'salisbury',
      state: 'cumbria',
      postcode: 'NN4 9LF',
      coordinates: {
        latitude: '-5.3269',
        longitude: '48.1334'
      },
      timezone: {
        offset: '+2:00',
        description: 'Kaliningrad, South Africa'
      }
    },
    email: 'nicky.vargas@example.com',
    login: {
      uuid: 'bee8fcb3-1261-47ff-ab6f-75c7be01077c',
      username: 'yellowladybug417',
      password: 'gogogo',
      salt: 'ixgD083Q',
      md5: '3c6400fffd6adddba4df6856079d61fd',
      sha1: '8f109908f4db48a9a8743ecf1ecc0d66e2536888',
      sha256: 'be9fd8363e25c08f58fb111600673a3c548b9f158fc847b5c48212f5fb2fa3c3'
    },
    dob: {
      date: '1986-01-13T11:28:23Z',
      age: 32
    },
    registered: {
      date: '2009-09-11T00:05:01Z',
      age: 9
    },
    phone: '016973 35595',
    cell: '0707-619-299',
    id: {
      name: 'NINO',
      value: 'TZ 01 18 38 X'
    },
    picture: {
      large: 'https://randomuser.me/api/portraits/women/69.jpg',
      medium: 'https://randomuser.me/api/portraits/med/women/69.jpg',
      thumbnail: 'https://randomuser.me/api/portraits/thumb/women/69.jpg'
    },
    nat: 'GB'
  }
]
