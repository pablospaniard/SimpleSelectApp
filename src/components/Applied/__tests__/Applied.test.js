import React from 'react'
import { render, cleanup } from 'react-testing-library'
import Applied from '../Applied'
import { persons } from '../../../mockForTests/persons'

describe('Applied should render properly', () => {
  afterEach(cleanup)

  it('should match snapshot', () => {
    const { container } = render(<Applied persons={persons} />)
    expect(container.firstChild).toMatchSnapshot('Applied_snapshot_1')
  })
})
