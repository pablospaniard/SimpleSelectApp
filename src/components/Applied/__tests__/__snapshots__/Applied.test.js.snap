// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Applied_snapshot_1 1`] = `
<table
  class="MuiTable-root-3 Applied-table-2"
>
  <thead
    class="MuiTableHead-root-4"
  >
    <tr
      class="MuiTableRow-root-5 MuiTableRow-head-8"
    >
      <th
        class="MuiTableCell-root-10 MuiTableCell-head-11"
        scope="col"
      >
        Foto
      </th>
      <th
        class="MuiTableCell-root-10 MuiTableCell-head-11"
        scope="col"
      >
        Full Name
      </th>
      <th
        class="MuiTableCell-root-10 MuiTableCell-head-11"
        scope="col"
      >
        Email
      </th>
      <th
        class="MuiTableCell-root-10 MuiTableCell-head-11"
        scope="col"
      >
        CellPhone
      </th>
      <th
        class="MuiTableCell-root-10 MuiTableCell-head-11"
        scope="col"
      >
        Nationality
      </th>
      <th
        class="MuiTableCell-root-10 MuiTableCell-head-11"
        scope="col"
      />
    </tr>
  </thead>
  <tbody
    class="MuiTableBody-root-18"
  >
    <tr
      class="MuiTableRow-root-5"
    >
      <td
        class="MuiTableCell-root-10 MuiTableCell-body-12"
      >
        <img
          src="https://randomuser.me/api/portraits/thumb/women/69.jpg"
          alt="foto"
        />
      </td>
      <th
        class="MuiTableCell-root-10 MuiTableCell-body-12"
        scope="row"
      >
        nicky vargas
      </th>
      <td
        class="MuiTableCell-root-10 MuiTableCell-body-12"
      >
        nicky.vargas@example.com
      </td>
      <td
        class="MuiTableCell-root-10 MuiTableCell-body-12"
      >
        0707-619-299
      </td>
      <td
        class="MuiTableCell-root-10 MuiTableCell-body-12"
      >
        GB
      </td>
      <td
        class="MuiTableCell-root-10 MuiTableCell-body-12"
      >
        <span
          class="MuiButtonBase-root-35 MuiIconButton-root-29 MuiSwitchBase-root-25 MuiCheckbox-root-19 MuiCheckbox-colorSecondary-24"
        >
          <span
            class="MuiIconButton-label-34"
          >
            <svg
              class="MuiSvgIcon-root-38"
              focusable="false"
              viewBox="0 0 24 24"
              aria-hidden="true"
              role="presentation"
            >
              <path
                d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
              />
            </svg>
            <input
              class="MuiSwitchBase-input-28"
              type="checkbox"
              data-indeterminate="false"
              value=""
            />
          </span>
          <span
            class="MuiTouchRipple-root-47"
          />
        </span>
      </td>
    </tr>
  </tbody>
</table>
`;
